common/src/main/java/org/conscrypt/ConscryptFileDescriptorSocket.java

```diff
@@ -40,6 +40,7 @@
 import javax.net.ssl.SSLException;
 import javax.net.ssl.SSLHandshakeException;
 import javax.net.ssl.SSLParameters;
+import javax.net.ssl.SSLPeerUnverifiedException;
```

Comment:
Unneeded import.

Reply:
Thanks! (Throwback to my first pass when I misunderstood the contract when this is called before the handshake completes)